docker/hoodie/hadoop/base/Dockerfile

```diff
@@ -22,7 +22,7 @@ USER root
 # Default to UTF-8 file.encoding
 ENV LANG C.UTF-8
 
-ARG HADOOP_VERSION=2.8.4 
+ARG HADOOP_VERSION=3.3.4 
```

Comment:
we should be able to simplify this without maintaining a base hadoop image. use official one https://hub.docker.com/r/apache/hadoop/tags

do this in compose.yml. use platform let arm machine enter compatibility mode:

```yaml
    hdfs-namenode:
        image: apache/hadoop:3.4
        platform: linux/amd64
```